ff/serving/runner/worker/worker.go

```diff
@@ -15,11 +16,29 @@ func CreateAndRun() error {
 	if !ok {
 		return errors.New("NAME not set")
 	}
-	runner, err := runner.Create(name, []byte(config))
+	jobRunner, err := runner.Create(name, []byte(config))
 	if err != nil {
 		return err
 	}
-	watcher, err := runner.Run()
+	indexString, hasIndexEnv := os.LookupEnv("JOB_COMPLETION_INDEX")
```

Comment:
Why isn't this part of config? Why jump through hoops to do the whole SetIndex thing

Reply:
It's based on one way of dealing with how Kubernetes delegates work to pods. Currently I'm using indexed completion which seperates the job into seperate pods and assigns each one an index in the environment variable job_completion_index. The alternative is to have a single pod with multiple containers, each one with different environment variables corresponding to their indices.

Doing it the single pod way would do away with this awkward convention since the index doesn't have to be separated from the config, but it requires either making multiple jobs or having all the runners  in one pod.